Tests/Tests/SDAnimatedImageTest.m

```diff
@@ -153,11 +153,7 @@ - (void)test12AnimatedImageViewLayerContents {
 #if SD_MAC
     expect(imageView.wantsUpdateLayer).beTruthy();
 #else
-#if TARGET_OS_MACCATALYST
-    // macOS's UIImageView seems does not setup layer.contents until render on screen
```

Comment:
I test again. In the final version iOS 13.0, the UIImage's layer.contents is not setup until it render on screen. So we do not need this test.
This does not effect the animation behavior of `SDAnimatedImageView`